pyramid_oereb/contrib/print_proxy/tocPages.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+import logging
+
+
+from PIL import ImageFont
+
+import textwrap
+import os
+
+log = logging.getLogger(__name__)
+
+class TocPages():
+    """
+    This class is inspired by this example: 
+    https://gist.github.com/turicas/1455973
+    """
+    FONT_CADASTRA_NORMAL =  os.path.dirname(os.path.abspath(__file__)) + '/fonts/Cadastra/Cadastra.ttf'
+    FONT_CADASTRA_BOLD = os.path.dirname(os.path.abspath(__file__)) + '/fonts/Cadastra/CadastraBd.ttf'
+    
+    def __init__(self,extract):
+        # variables taken from template toc.jrxml
+        self.disposable_height = 842 - 70
+        self.d1_height = 77
+        self.d2_height = 29
+        self.d3_height = 61
+        self.d4_height = 44
+        self.d5_height = 15
+        self.d6_height = 90
+        self.d6_right_height = 23
+        self.d6_right_width = 233
+        self.d6_stuff_y_location = 39
+        self.d6_left_height = 0  # FIXME: compute this
+        self.title_size = 62
+        self.toc_title_height=15 + 62 + 12 # height + location + item starting position
+        self.toc_item_height=20
+        self.not_concerned_themes_title_height=15 + 26 # height + location
+        self.not_concerned_themes_item_height=12
+        self.theme_without_data_title_height=12
+        self.theme_without_data_item_height=12
+        self.extract = extract
+        self.total_length = self.compute_total_lenght()
+    
+    def get_text_size(self, font_filename, font_size, text):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/pyramid_oereb/pyramid_oereb/pullRequest?prid=4756827)